packages/experimental-utils/CHANGELOG.md

```diff
@@ -3,1287 +3,3 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-# [5.4.0](https://github.com/typescript-eslint/typescript-eslint/compare/v5.3.1...v5.4.0) (2021-11-15)
```

Comment:
I'm not sure about this change to just remove the content.

Every time we publish packages, we publish all of them based on the unified version, regardless of if there were changes, so Lerna will continue to write to this file for as long as we have the package around.

Because we won't be committin changes to the `experimental-utils` scope once this PR goes in, all the Lerna updates will look like this one do:

```
## [5.3.1](https://github.com/typescript-eslint/typescript-eslint/compare/v5.3.0...v5.3.1) (2021-11-08)

**Note:** Version bump only for package @typescript-eslint/experimental-utils
```

I _think_ Lerna will keep the paragraph at the top intact even if we make changes to it manually, so I think the best way forward would be to restore the previous content (because at the end of the day it is still an accurate changelog for this package), and add a clear disclaimer at the top pointing people towards `utils` instead.